.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ references:
   android_config: &android_config
     working_directory: ~/work
     docker:
-      - image: circleci/android:api-30
+      - image: cimg/android:2022.03.1
```

Comment:
Made this update to Circle's new image as it was giving me warnings when CI ran.